sql/parse/parse.go

```diff
@@ -2034,8 +2080,7 @@ func tableNameToUnresolvedTableAsOf(tableName sqlparser.TableName, asOf sql.Expr
 	return plan.NewUnresolvedTableAsOf(tableName.Name.String(), tableName.Qualifier.String(), asOf)
 }
 
-func convertAlterIndex(ctx *sql.Context, ddl *sqlparser.DDL) (sql.Node, error) {
-	table := tableNameToUnresolvedTable(ddl.Table)
```

Comment:
i have a PR that rewrites this file, I'd appreciate undoing unnecessary changes that might amplify merge conflicts

Reply:
Whoops, fixed.